lib/timescaledb/rails/railtie.rb

```diff
@@ -6,13 +6,15 @@ module Timescaledb
   module Rails
     # :nodoc:
     class Railtie < ::Rails::Railtie
-      initializer 'timescaledb-rails.require_timescale_models' do
+      config.record_base = '::ActiveRecord::Base'
+
+      config.to_prepare do
```

Comment:
need to use `to_prepare` so integrating apps have a change to customize the record base

Reply:
Would be ok to revert this change now?